src/scripts/cp.js

```diff
@@ -1220,6 +1220,27 @@ InteractiveBoard.prototype.attachElement = function (
       }
     }
 
+    if (
+      element &&
+      element.action !== undefined &&
+      element.action.library.substr(0, 13) === 'H5P.NDLAShape'
+    ) {
```

Comment:
I think it's safe to use optional chaining here, as we use Babel to transform modern code to legacy.

```suggestion
    const isNDLAShape = element?.action?.library.substr(0, 13) === 'H5P.NDLAShape';
    if (isNDLAShape) {
```